docs/src/user_guide/type_structure.md

```diff
@@ -0,0 +1,16 @@
+```@meta
```

Comment:
I'm not familiar with Literate.  Is it possible to avoid add this generated file to the repository?

Reply:
resolved in f20b2df